website/package.json

```diff
@@ -41,17 +41,21 @@
     "alex": "^8.0.0",
     "docusaurus": "1.14.5",
     "highlight.js": "^9.15.10",
-    "remarkable": "^2.0.0"
+    "remarkable": "^2.0.0",
+    "tokenize-comment": "^3.0.1"
```

Comment:
Why `tokenize-comment` is located under `dependencies` instead of `devDependencies`? Did I miss something?

Reply:
I agree, this can be a dev dependency like the rest of the doc generation script's dependencies. I'm not sure there's a strong distinction though, given the way the website is deployed at the moment (e.g. does the `alex` linter need to be under `dependencies`?)

Reply:
> (e.g. does the alex linter need to be under dependencies?)

That's a good question, I'm going to check this and and based on the results I will adjust the #2140 PR. Alex has been added almost a year ago when I was not an active contributor.